protocols/kad/src/lib.rs

```diff
@@ -72,7 +48,10 @@ pub mod record {
 
 mod addresses;
 mod behaviour;
+mod handler;
 mod jobs;
+mod kbucket;
```

Comment:
Why is it not public ?
take a look into get_closest_local_peers signature, it has a kbucket::Key as a parameter

Reply:
and the same story for handler + ConnectionClosed event

Reply:
`Key` is publicly exposed as `KBucketKey`. Does that solve your issue? The handler is an implementation detail. If you need to name it, you can use `<Kademlia as NetworkBehaviour>::ConnectionHandler`.

Reply:
yep, sound good, thanks